plugins/resources/src/structfile.cpp

```diff
@@ -2287,7 +2287,7 @@ irods::error bundle_cache_dir( int                _index,
 
         // =-=-=-=-=-=-=-
         // set the format of the tar archive
-        archive_write_set_format_ustar( arch );
+        archive_write_set_format_pax( arch );
 
```

Comment:
Clang-format appears to be upset about the spaces in the parentheses and the trailing newline. I vote that we address it. Thoughts?

Reply:
I'll leave that up to others.  If we change it it will mean that there are inconsistencies around calling `archive_write_set_format*` methods in that file.

Reply:
Those kinds of inconsistencies are okay _(I'm assuming you're referring to some calls containing spaces and other not?)_.

Let's apply clang-format.

Reply:
Done